### PR TITLE
rddepman: bump WSLDistro from 0.32 to 0.33

### PR DESCRIPTION
**Pull request by rancherbot**

### CODE REVIEW

File: src/go/nerdctl-stub/nerdctl_commands_generated.go [hunk 1]
```diff
@@ -158,12 +157,41 @@ var commands = map[string]commandDefinition{
...
 
+	"builder build": {
+		commandPath: "builder build",
+		subcommands: map[string]struct{}{},
+		options: map[string]argHandler{
+			"--build-arg":     ignoredArgHandler,
+			"--buildkit-host": ignoredArgHandler,
+			"--cache-from":    ignoredArgHandler,
+			"--cache-to":      ignoredArgHandler,
+			"--file":          ignoredArgHandler,
```
Comment (mook-as) [inline src/go/nerdctl-stub/nerdctl_commands_generated.go:175]: Shouldn't this have an override (at the bottom of `parse_args.go`)?

File: src/go/nerdctl-stub/nerdctl_commands_generated.go [hunk 2]
```diff
@@ -158,12 +157,41 @@ var commands = map[string]commandDefinition{
...
+			"--iidfile":       ignoredArgHandler,
```
Comment (mook-as) [inline src/go/nerdctl-stub/nerdctl_commands_generated.go:176]: Same  here… and probably some more.

**Review by mook-as — CHANGES_REQUESTED** [review]

File: src/go/nerdctl-stub/nerdctl_commands_generated.go (hunk 2 above)
Comment (mook-as) [reply src/go/nerdctl-stub/nerdctl_commands_generated.go:176]: We'd need to write more `*ArgHandler` functions, basically. I'll deal with it.

Comment (ericpromislow) [reply src/go/nerdctl-stub/nerdctl_commands_generated.go:176]: In this PR or a future one?

Comment (jandubois) [reply src/go/nerdctl-stub/nerdctl_commands_generated.go:176]: > In this PR or a future one?

Mark has already written them in this PR; that's what all the code changes are about. The rddepman commit just bumps the version number for wsl-distro. Everything else is about dealing with new options that take file/directory name arguments.

**Review by ericpromislow — APPROVED** [review]
LGTM. Tests pass. Some randomly selected new options are accepted, and non-options are reported and rejected.

**Pull request merged**